src/System.IO.Compression/src/Resources/Strings.resx

```diff
@@ -154,10 +154,10 @@
     <value>Stream does not support writing.</value>
   </data>
   <data name="ObjectDisposed_StreamClosed" xml:space="preserve">
-    <value>Can not access a closed Stream.</value>
+    <value>Cannot access a closed stream.</value>
   </data>
   <data name="UnknownBlockType" xml:space="preserve">
-    <value>Unknown block type. Stream might be corrupted.</value>
+    <value>Unknown block type. stream might be corrupted.</value>
```

Comment:
stream should be capitalized as it starts a sentence.

Reply:
aahhh sorry